packages/service-library/src/servicelib/long_running_tasks/task.py

```diff
@@ -56,11 +63,29 @@ def __name__(self) -> str: ...
 
 
 class TaskRegistry:
-    REGISTERED_TASKS: ClassVar[dict[RegisteredTaskName, TaskProtocol]] = {}
+    REGISTERED_TASKS: ClassVar[
```

Comment:
make it protected? `_REGISTERED_TASKS`

Reply:
yes, good one, this forgot to change this